src/DynamoCoreWpf/ViewModels/Watch3D/HelixWatch3DViewModel.cs

```diff
@@ -1162,26 +1164,35 @@ private void SetSelection(IEnumerable items, bool isSelected)
 
         private static void SetDepthBiasBasedOnSelection(bool isSelected, IEnumerable<Element3D> element3Ds)
         {
-            //selected should be lowest depth
-            var newDepth = DepthBiasSelectedMesh;
             foreach (var element in element3Ds)
             {
-                if (element is DynamoGeometryModel3D geom)
+                //selected should be lowest depth
+                var stdbias = 0;
+                var newDepth = 0;
+                switch (element)
+                {
+                    case DynamoGeometryModel3D t1:
+                        stdbias = DepthBiasMesh;
+                        break;
+                    case DynamoLineGeometryModel3D t1:
+                        stdbias = DepthBiasLine;
+                        break;
+                    case DynamoPointGeometryModel3D t1:
+                        stdbias = DepthBiasPoint;
+                        break;
+                    // if this is an unknown type, don't modify depth bias.
+                    default:
+                        return;
+                }
+                //selected bias
+                newDepth = stdbias - DepthBiasSelectedOffset;
```

Comment:
Does this mean that for points, it will be negative? Is that a valid depth bias?

Reply:
yes - this is a good question - in addition to the `DepthBias` there is also a `DepthBiasClamp` - 
https://docs.microsoft.com/en-us/windows/win32/direct3d11/d3d10-graphics-programming-guide-output-merger-stage-depth-bias

the docs seem to indicate that if the clamp is set to 0 - which ours should be unless helix does has some other default, that it's not used.

then the final value is calculated like:
```
if ( (DepthBias != 0) || (SlopeScaledDepthBias != 0) )
    z = z + Bias
```
so this seems to be valid.